src/Winium.Desktop.Driver/CommandExecutors/NewSessionExecutor.cs

```diff
@@ -32,6 +31,14 @@ protected override string DoImpl()
             // Gives sometime to load visuals (needed only in case of slow emulation)
             Thread.Sleep(this.Automator.ActualCapabilities.LaunchDelay);
 
+            // Update running application process pointing to the real process instead of the launcher in such cases
+            if ((this.Automator.Application.HasExited()) && (this.ExecutedCommand.Parameters["desiredCapabilities"]["realProcessName"] != null))
```

Comment:
We can do this to avoid getting param 2 times
if ((this.Automator.Application.HasExited()))
{
    var realProcessName = this.ExecutedCommand.Parameters["desiredCapabilities"]
    if(realProcessName != null){
        // Update launched process by process name if it's exited
        this.Automator.Application.UpdateProcessByName(realProcessName.ToString());
    }
}